README.rst

```diff
@@ -36,58 +35,65 @@ See `edx/configuration <http://github.com/edx/configuration>`_ for instructions
 
 Configuration
 -------------
+1. Install requirements:
+
+.. code:: bash
+
+    pip install -r requirements/base.txt
+
+2 - Install edx platform pages:
+
+.. code:: bash
+
+    paver install_pages
 
-Edit the configuration file ``config.ini`` to provide information about the system under test.
-You can specify another configuration file by setting the ``CONFIG_PATH`` environment variable.
 
 
 Running Tests Locally
 ---------------------
 
 Within the Vagrant environment, the tests are installed in /opt/dev/edx-e2e-tests,
```

Comment:
@raeeschachar There are two possible methods of configuring the repo. One within an existing vagrant machine e.g edx-platform devstack and other within a standalone vagrant machine for edx-e2e-tests. The documentation is about configuring the repo within a standalone vagrant machine. What are you suggesting of doing?